examples/plugins/iowait/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func main() {
+	hostname, _ := os.Hostname()
+	var (
+		addr   = flag.String("addr", "/var/run/scope/plugins/iowait.sock", "unix socket to listen for connections on")
+		hostID = flag.String("hostname", hostname, "hostname of the host running this plugin")
+	)
+	flag.Parse()
+
+	log.Println("Starting...")
+
+	// Check we can get the iowait for the system
+	_, err := iowait()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	os.Remove(*addr)
```

Comment:
Same comment about providing a library to deal with registration and the handshake. I understand we may not want to do that now. In that case can you create an issue?